CMakeLists.txt

```diff
@@ -387,29 +387,26 @@ endforeach()
 
 # Find Lua interpreter
 include(LuaHelpers)
-set(LUA_DEPENDENCIES lpeg mpack bit)
 if(NOT LUA_PRG)
   foreach(CURRENT_LUA_PRG luajit lua5.1 lua5.2 lua)
     # If LUA_PRG is set find_program() will not search
     unset(LUA_PRG CACHE)
-    unset(LUA_PRG_WORKS)
     find_program(LUA_PRG ${CURRENT_LUA_PRG})
 
     if(LUA_PRG)
-      check_lua_deps(${LUA_PRG} "${LUA_DEPENDENCIES}" LUA_PRG_WORKS)
```

Comment:
Yes, that's how it used to work.  `LUA_PRG_WORKS` would only be truthy if both the lua binary existed and the required modules were available.  For example, if luajit was present but missing a required module, this loop would iterate again and find the installed lua5.1 with all the required modules.